src/test/java/com/hocs/test/glue/security/SecurityStepDef.java

```diff
@@ -330,5 +330,88 @@ public String monthOfDate(String date) {
     public String yearOfDate(String date) {
         return date.split("/")[2];
     }
+
+    @And("I create a DCU case for security testing")
```

Comment:
Can we add DTEN in here so its "I create a DCU DTEN case...", an explicit case type will make it easier for future maintenance.